crates/driver/openapi.yml

```diff
@@ -331,7 +331,7 @@ components:
             Information about tokens used in the auction.
         deadline:
           $ref: "#/components/schemas/DateTime"
-        surplusCapturingJitOrderOwners:
+        cowAmms:
```

Comment:
Breaking change, we need to tell the collocated solvers about this before launching :warning: